examples/jesuMusic/jesuMusic.zig

```diff
@@ -0,0 +1,868 @@
+const gba = @import("gba");
+const Enable = gba.utils.Enable;
+
+export const gameHeader linksection(".gbaheader") = gba.initHeader("JESUMUSIC", "AJME", "00", 0);
+
+// Must be aligned or else memcpy will copy a byte at a time,
+// and VRAM doesn't like that.
+// https://github.com/Games-by-Mason/SPIRV-Reflect-zig/pull/1#issuecomment-2655358098
+/// File contains tile image data.
+const charset_data align(4) = @embedFile("charset.bin").*;
+
+const hex_digits: [16]u8 = .{
+    '0', '1', '2', '3', '4', '5', '6', '7',
+    '8', '9', 'A', 'B', 'C', 'D', 'E', 'F',
+};
+
+const pitch_names: [84][]const u8 = .{
+    "C-2", "C#2", "D-2", "D#2", "E-2", "F-2",
+    "F#2", "G-2", "G#2", "A-2", "A#2", "B-2",
+    "C-3", "C#3", "D-3", "D#3", "E-3", "F-3",
+    "F#3", "G-3", "G#3", "A-3", "A#3", "B-3",
+    "C-4", "C#4", "D-4", "D#4", "E-4", "F-4",
+    "F#4", "G-4", "G#4", "A-4", "A#4", "B-4",
+    "C-5", "C#5", "D-5", "D#5", "E-5", "F-5",
+    "F#5", "G-5", "G#5", "A-5", "A#5", "B-5",
+    "C-6", "C#6", "D-6", "D#6", "E-6", "F-6",
+    "F#6", "G-6", "G#6", "A-6", "A#6", "B-6",
+    "C-7", "C#7", "D-7", "D#7", "E-7", "F-7",
+    "F#7", "G-7", "G#7", "A-7", "A#7", "B-7",
+    "C-8", "C#8", "D-8", "D#8", "E-8", "F-8",
+    "F#8", "G-8", "G#8", "A-8", "A#8", "B-8",
+};
```

Comment:
Pitches well outside what is used in this track are included as an illustration and so that this example could be used as a starting point for other audio projects.